game-core/src/main/java/games/strategy/engine/framework/startup/ui/posted/game/pbf/ForumPosterEditorViewModel.java

```diff
@@ -62,6 +71,17 @@ void setForumPassword(final char[] password) {
     readyCallback.run();
   }
 
+  private boolean isDummyPassword(final char[] password) {
+    boolean dummyPassword = true;
+    for (final Character passwordChar : password) {
+      if (passwordChar != '*') {
+        dummyPassword = false;
+        break;
+      }
+    }
+    return password.length > 0 && dummyPassword;
```

Comment:
```suggestion
    return password.length > 0 && Arrays.stream(password).allMatch(c -> c == '*');
```

Reply:
Arrays.stream() kept doing a `Stream<char[]>` instead of `Stream<Character>`, I'll be curious if this works.

Reply:
You're right forgot this detail, however I found this approach:
```
CharBuffer.wrap(abc).chars().allMatch(c -> c == '*')
```